### PR TITLE
Add information about the dbt-decodable adapter 

### PR DESCRIPTION
**Pull request by matsukawa-maria**
## Checklist

- [X] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [X] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."
- [x] Needs technical review

Adding new pages (delete if not applicable):
- [X] Add page to `website/sidebars.js`
- [X] Provide a unique filename for the new page

### CODE REVIEW

**Comment by mirnawong1** [comment]
hey @matsukawa-maria thanks for opening this up and we'll take a look at this pr! 

hey @dataders - is there anything, in particular, we should do for new set up files like this?

**Review by rmetzger — DISMISSED** [review]
From the dbt adapter's perspective, this looks good to me for an initial version

File: website/docs/reference/warehouse-setups/decodable-setup.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+
+
+<b>cleanup(list, models, seeds, tests)</b>
+<ul>
+<li>list: An optional list of resource entity names to delete. Defaults to none.</li>
+<li>models: Specify whether to include models during cleanup. Defaults to true.</li>
+<li>seeds: Specify whether to include seeds during cleanup. Defaults to true.</li>
+<li>tests: Specify whether to include tests during cleanup. Defaults to true.</li>
+</ul>
+Delete all Decodable entities resulting from the materialization of the project’s resources, i.e. connections, streams and pipelines.
```
Comment (mirnawong1) [inline website/docs/reference/warehouse-setups/decodable-setup.md]:
```suggestion
<br /><br />Delete all Decodable entities resulting from the materialization of the project’s resources, i.e. connections, streams, and pipelines.
```

File: website/docs/reference/warehouse-setups/decodable-setup.md [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+
+Some core functionality may be limited. If you're interested in contributing, see the source code for the repository listed below.
+:::
+
+<h2> Overview of {frontMatter.meta.pypi_package} </h2>
+
+<ul>
+    <li><strong>Maintained by</strong>: {frontMatter.meta.maintained_by}</li>
+    <li><strong>Authors</strong>: {frontMatter.meta.authors}</li>
+    <li><strong>GitHub repo</strong>: <a href={`https://github.com/decodableco/dbt-decodable}`}>{frontMatter.meta.github_repo}</a><a href={`https://github.com/${frontMatter.meta.github_repo}`}><img src={`https://img.shields.io/github/stars/${frontMatter.meta.github_repo}?style=for-the-badge`}/></a></li>
```
Comment (mirnawong1) [inline website/docs/reference/warehouse-setups/decodable-setup.md:29]:
```suggestion
    <li><strong>GitHub repo</strong>: <a href={`https://github.com/decodableco/dbt-decodable}`}>{frontMatter.meta.github_repo}</a><a href={`https://github.com/${frontMatter.meta.github_repo}`}><img src={`https://img.shields.io/github/stars/${frontMatter.meta.github_repo}?style=for-the-badge`}/></a></li>
```

File: website/docs/reference/warehouse-setups/decodable-setup.md [hunk 3]
```diff
@@ -0,0 +1,156 @@
+---
+title: "Decodable setup"
+id: "decodable-setup"
+meta:
+  maintained_by: Decodable
+  authors: "Decodable Team"
+  github_repo: 'https://github.com/decodableco/dbt-decodable'
```
Comment (mirnawong1) [inline website/docs/reference/warehouse-setups/decodable-setup.md]:
```suggestion
  github_repo: '/decodableco/dbt-decodable'
```

File: website/docs/reference/warehouse-setups/decodable-setup.md [hunk 4]
```diff
@@ -0,0 +1,156 @@
+---
+title: "Decodable setup"
+id: "decodable-setup"
+meta:
+  maintained_by: Decodable
+  authors: "Decodable Team"
+  github_repo: '/decodableco/dbt-decodable'
```
Comment (mirnawong1) [inline website/docs/reference/warehouse-setups/decodable-setup.md]:
```suggestion
  github_repo: 'decodableco/dbt-decodable'
```

File: website/docs/reference/warehouse-setups/decodable-setup.md [hunk 5]
```diff
@@ -0,0 +1,156 @@
+---
+title: "Decodable setup"
+id: "decodable-setup"
+meta:
+  maintained_by: Decodable
+  authors: "Decodable Team"
+  github_repo: 'decodableco/dbt-decodable'
```
Comment (mirnawong1) [inline website/docs/reference/warehouse-setups/decodable-setup.md:7]:
```suggestion
  github_repo: ['decodableco/dbt-decodable'](https://github.com/decodableco/dbt-decodable)
```

File: website/docs/reference/warehouse-setups/decodable-setup.md [hunk 6]
```diff
@@ -0,0 +1,156 @@
+---
+title: "Decodable setup"
+id: "decodable-setup"
+meta:
+  maintained_by: Decodable
+  authors: "Decodable Team"
+  github_repo: ['decodableco/dbt-decodable'](https://github.com/decodableco/dbt-decodable)
```
Comment (mirnawong1) [inline website/docs/reference/warehouse-setups/decodable-setup.md]:
```suggestion
  github_repo: '[decodableco/dbt-decodable](https://github.com/decodableco/dbt-decodable)'
```

File: website/docs/reference/warehouse-setups/decodable-setup.md [hunk 7]
```diff
@@ -0,0 +1,156 @@
+---
+title: "Decodable setup"
+id: "decodable-setup"
+meta:
+  maintained_by: Decodable
+  authors: "Decodable Team"
+  github_repo: '[decodableco/dbt-decodable](https://github.com/decodableco/dbt-decodable)'
```
Comment (mirnawong1) [inline website/docs/reference/warehouse-setups/decodable-setup.md]:
```suggestion
  github_repo: [decodableco/dbt-decodable](https://github.com/decodableco/dbt-decodable)
```

File: website/docs/reference/warehouse-setups/decodable-setup.md [hunk 8]
```diff
@@ -0,0 +1,156 @@
+---
+title: "Decodable setup"
+id: "decodable-setup"
+meta:
+  maintained_by: Decodable
+  authors: "Decodable Team"
+  github_repo: [decodableco/dbt-decodable](https://github.com/decodableco/dbt-decodable)
```
Comment (mirnawong1) [inline website/docs/reference/warehouse-setups/decodable-setup.md]:
```suggestion
  github_repo: 'decodableco/dbt-decodable'
```

**Review by mirnawong1 — APPROVED** [review]
hey @matsukawa-maria thanks for opening this up and made some tweaks to the git URL as it was returning a 'page not found'. 

will go ahead and merge this for you!

**Comment by rmetzger** [comment]
🎉 thanks a lot!

**Comment by matsukawa-maria** [comment]
Thank you!

**Pull request merged**